ai/yolo/auto_labeling_script/main.py

```
@@ -0,0 +1,206 @@
+                cropped_image_path=crop_path,
+                output_txt_path=txt_path,
+                class_id=class_id,
+                preview_path=preview_path,
+                min_size=min_size,
+                max_size=max_size,
+                step=step,
+                min_score=min_score
+            )
+            print(f"[OK] {filename} -> score={result['score']:.4f}")
+            ok_count += 1
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)